src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -481,15 +481,18 @@ public function newInstance($attributes = array(), $exists = false)
 	 * Create a new model instance that is existing.
 	 *
 	 * @param  array  $attributes
+	 * @param  string  $connection
```

Comment:
Lol. I just can't see the real benefit to going through ensuring all old docblocks have a `|null` in them where they should have one. Time is precious, and should be used to it's maximum potential. :)